shelley/chain-and-ledger/executable-spec/src/LedgerState.hs

```diff
@@ -610,8 +612,8 @@ enoughWits
   -> Dms dsignAlgo
   -> UTxOState hashAlgo dsignAlgo
   -> Validity
-enoughWits tx@(Tx _ wits _) d u =
-  if witsVKeyNeeded (u ^. utxo) tx d `Set.isSubsetOf` signers
+enoughWits tx@(Tx _ wits _) d' u =
+  if witsVKeyNeeded (u ^. utxo) tx d' `Set.isSubsetOf` signers
```

Comment:
why the rename, is there a name clash?

Reply:
yes, I imported the protocol parameter `d` for the implementation of the overlay schedule.